lib/droplet_kit.rb

```diff
@@ -69,7 +69,7 @@ module DropletKit
   autoload :CertificateResource, 'droplet_kit/resources/certificate_resource'
   autoload :FirewallResource, 'droplet_kit/resources/firewall_resource'
   autoload :CDNResource, 'droplet_kit/resources/cdn_resource'
-  autoload :KubernetesResource, 'droplet_kit/resources/kubernetes_resource'
+  autoload :KubernetesClustersResource, 'droplet_kit/resources/kubernetes_clusters_resource'
```

Comment:
Should this be `KubernetesClusterResource`, following the singular format of rest of the resources?